### PR TITLE
EVM: native runner cleanup

### PR DESCRIPTION
**Pull request by shaopengw**

### CODE REVIEW

File: modules/evm/src/runner/native.rs [hunk 1]
```diff
@@ -364,6 +319,29 @@ impl<'vicinity, 'config, T: Trait> Handler<'vicinity, 'config, T> {
...
 
+/// Create `try_or_fail` and `try_or_rollback`.
+macro_rules! create_try {
+	( $map_err:expr ) => {
+		macro_rules! try_or_fail {
+			( $e:expr ) => {
+				match $e {
+					Ok(v) => v,
+					Err(e) => return Capture::Exit($map_err(e)),
+					}
```
Comment (xlc) [inline modules/evm/src/runner/native.rs:330]: indent is wrong

File: modules/evm/src/runner/native.rs [hunk 2]
```diff
@@ -364,6 +319,29 @@ impl<'vicinity, 'config, T: Trait> Handler<'vicinity, 'config, T> {
 	}
 }
 
+/// Create `try_or_fail` and `try_or_rollback`.
+macro_rules! create_try {
+	( $map_err:expr ) => {
+		macro_rules! try_or_fail {
+			( $e:expr ) => {
```
Comment (xlc) [inline modules/evm/src/runner/native.rs:326]: why this needs to be macro? can we make it generic function?

Comment (shaopengw) [reply modules/evm/src/runner/native.rs:326]: Generic function cannot cover the requirement that "if err, return a non err value, else do nothing". Usually `try!` and `?` syntax could handle this scenario but the handler functions doesn't return `Result`.

File: modules/evm/src/runner/native.rs (hunk 1 above)
Comment (shaopengw) [reply modules/evm/src/runner/native.rs:330]: I did it right, and then cargo fmt made it wrong.😂

File: modules/evm/src/runner/native.rs (hunk 2 above)
Comment (xlc) [reply modules/evm/src/runner/native.rs:326]: ok. we can refactor the code so it runs in a method return `Result` later

**Review by xlc — APPROVED** [review]

**Pull request merged**